modules/twinklewarn.js

```diff
@@ -168,34 +168,44 @@ Twinkle.warn.messages = {
 	level1: {
 		"Common warnings": {
 			"uw-vandalism1": {
-				label: "Vandalism",
+				label: "Unconstructive editing",
 				summary: "General note: Unconstructive editing"
 			},
 			"uw-disruptive1": {
 				label: "Disruptive editing",
-				summary: "General note: Unconstructive editing"
+				summary: "General note: Disruptive editing"
 			},
 			"uw-test1": {
 				label: "Editing tests",
 				summary: "General note: Editing tests"
 			},
 			"uw-delete1": {
-				label: "Removal of content, blanking",
-				summary: "General note: Removal of content, blanking"
+				label: "Removal of content without adequate explanation",
+				summary: "General note: Removal of content without adequate explanation"
 			}
 		},
-		"Behavior in articles": {
-			"uw-biog1": {
-				label: "Adding unreferenced controversial information about living persons",
-				summary: "General note: Adding unreferenced controversial information about living persons"
+		"Vandalism and disruptive editing": {
+			"uw-vandalism1": {
+				label: "Unconstructive editing",
+				summary: "General note: Unconstructive editing"
 			},
-			"uw-defamatory1": {
-				label: "Addition of defamatory content",
-				summary: "General note: Addition of defamatory content"
+			"uw-subtle1": {
+				label: "Subtle vandalism",
+				summary: "General note: Subtle vandalism"
```

Comment:
While okay in the Twinkle menu, "vandalism" should not appear in the edit summary for level 1

Reply:
I'd fix this after merging instead of waiting for a fix here.